src/main/java/clients/ActivitiesClient.java

```diff
@@ -284,4 +283,37 @@ public void insertActivityWithAlterations(String body, List<JsonPathAlteration>
     public void deleteActivity(String id) {
         AwsUtil.deleteActivityById(id);
     }
+
+    private Response callOpenVisitCheck(String activityType, String testerStaffId) {
```

Comment:
Just a query, is the activityType value implemented across these methods ever used? 

Looking at the /activities/open endpoint, it only ever uses a testerStaffId in the lookup to dynamoDB.

Reply:
Removed, as indeed not really necessary.